basicsr/models/archs/sync_batchnorm/batchnorm.py

```diff
@@ -0,0 +1,416 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why not use the official syncbn? May try to use the official sync bn

Reply:
According to [pytorch doc](https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html) SyncBatchNorm only supports DDP training, so I cannot test it now. I shall settle with InstanceNorm2D for now.